src/tests/test_connection.py

```diff
@@ -401,3 +402,29 @@ def test_new_connection_with_alias(ip_empty, old_alias, new_alias):
         assert connection
         assert connection.url == "duckdb://"
         assert connection == connection.current
+
+
+def test_set_with_display_module(ip_empty, capsys):
+    ip_empty.run_cell("%load_ext sql")
+    ip_empty.run_cell("%sql duckdb:// --alias one")
+    ip_empty.run_cell("%sql duckdb:// --alias two")
+    ip_empty.run_cell("%sql one")
+
+    message = display.Message("Switching to connection one")
+    captured = capsys.readouterr()
+
+    assert str(message) in captured.out
```

Comment:
instead of checking "str in captured.out", check `str == captured.out`. this a more precise test because we also want to ensure we're not printing stuff that we shouldn't. e.g., if we print `Switching to connection one` 100 times this tests will still pass